src/pages/SearchDetails/components/SearchItemPC.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from 'pages/SearchDetails/SearchDetails.module.scss';
+import { Link } from 'react-router-dom';
+import { getMockItem, SHOPS } from '../static/mockup';
+
+interface Props {
+  shop: {
+    address: string,
+    placeName: string,
+    shopId: number,
+  },
+}
+
+export default function SearchItemPC({ shop }: Props) {
+  const { placeName, address } = shop;
+  const {
+    imageAlt, defaultImage, phoneNumber, image,
+  } = getMockItem();
+  return (
+    <Link to="/" className={styles.item}>
+      <div className={styles.image}>
+        <img className={styles.image__main} alt={imageAlt} src={defaultImage} />
```

Comment:
실제 데이터를 쓰게 된다면 defaultImage를 조건부로 쓰게 되겠죠?
목업데이터라면 구분해두는게 좋아보이네요.
```suggestion
        <img className={styles.image__main} alt={imageAlt} src={shop.image ?? defaultImage} />
```

근데 또 아래에는 잘 돼있네요? ㅋㅋ

Reply:
수정했습니다.
[5a10598](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/54/commits/5a10598f40b0eefd59cac85fb4aa9b7b6fbc8695)